loops.js

```diff
@@ -11,8 +11,9 @@
 // Dit heb je nog niet geleerd, maar bekijk hiervoor dit MDN-artikel eens: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/repeat
 // ==========================================
 
-
-
+for (let i = 0; i < 6; i++) {
+    console.log(`${ '*'.repeat(i)}`);
```

Comment:
Een template literal is hier overbodig. console.log('*'.repeat(i)) is voldoende. Verder wordt er bij de eerste log helemaal niets weergeven, dat komt omdat i gelijk is aan 0 in het begin. 

Je had dus bij het declareren van de i de waarde op 1 kunnen zetten, zoals dit: let i = 1. Of je had bij repeat i + 1 kunnen doen, zoals dit: console.log('*'.repeat(i + 1))